### DISCUSSION TITLE
Tooltip 位置不对

### DISCUSSION THREAD

**Discussion by ekuku1984** [ant-design/ant-design#45633] [General]
![image](https://github.com/ant-design/ant-design/assets/6016942/47e39b10-3408-4e84-8a20-b87e8da40804)

Tooltip 把 Slider 圆点都给档住了。

随便包含一个元素， 位置都不太对

看了官网的没问题，今天也升级到了 ANTD 最新版本了，还是不对， 项目内容比较多，以为是什么内容或样式冲突引起的。

然后在项目的入口把所有的 CSS 引入去掉， 就正常了， 最终发现是引入了 antd/dist/reset.css 引起的， 经过对 reset.css 文件的删减，发现是
body {
    margin: 0
}

这个样式引起的， 去掉 margin: 0 位置就正常了。

看了官网 body 也设置了 margin 又没事，不知道是原因了

**Comment by afc163** [comment]
给个重现看看？

**Reply by ekuku1984** [reply]
重现出来了， 看看

**Comment by ekuku1984** [comment]
![image](https://github.com/ant-design/ant-design/assets/6016942/ae090b6d-fe5f-40a1-827b-22d15df36a9b)

终于给重现出来了， 只要在 slider 的上级窗口添加样式宽高等于 100vw, 100vh, 就会有这种情况了

**Reply by afc163** [reply]
要不把链接发出来？

**Reply by ekuku1984** [reply]
搞了一天才把这个示例给搞出来， codesandbox.io 太慢了， 而 github 又经常打不开， 打不打得开全靠运气

https://codesandbox.io/s/competent-perlman-l8yryx?file=/src/App.js

**Reply by yoyo837** [reply]
挂个梯子的, 必备

**Comment by ekuku1984** [comment]
@afc163 有没有时间看一下这个问题

**Reply by Wxh16144** [reply]
我打开了你提供的地址，并没有复现😭

**Reply by afc163** [reply]
我也没有重现，你的浏览器版本和操作系统分别是什么？

<img width="676" alt="图片" src="https://github.com/ant-design/ant-design/assets/507615/960be82e-69b6-45cb-a2cb-6d22e942b690">


https://github.com/ant-design/ant-design/issues/45202#issuecomment-1803248958

**Reply by ekuku1984** [reply]
我的环境是 windows 11, EDGE 最新版， google chrome 最新版都试了， 都有问题

**Reply by Wxh16144** [reply]
确实有点问题，我帮你转为 issue 吧~
